pkgs/top-level/python-packages.nix

```diff
@@ -11914,12 +11914,12 @@ in {
   guessit = callPackage ../development/python-modules/guessit { };
 
   rebulk = buildPythonPackage rec {
```

Comment:
This has already been done and submitted in commit a9aefdeac9a8444b53931233b86f998561a9c89d by you.

I'll close this pull request.